### PR TITLE
Cli flag format

### LINKED ISSUE #727: [CLI v1] Change flags format
At the moment, all flags are in the snake case format like `--num_of_addresses` and should be all changed to the standard one, ie `--num-of-addresses`

### PR DESCRIPTION
**Pull request by sekulicd**
This formats CLI flags to standard format with "-" insteam "_".

This closes #727 

@altafan please review.

### CODE REVIEW

**Review by altafan — CHANGES_REQUESTED** [review]
Let's drop `market claim`, `fee claim` and `market reportfee` commands since they are deprecated.

Let's rename the flag `--millisatsperbyte` to `--millisats-per-byte` for all occurrences.

The rest LGTM.

**Comment by sekulicd** [comment]
> Let's drop `market claim`, `fee claim` and `market reportfee` commands since they are deprecated.
> 
> Let's rename the flag `--millisatsperbyte` to `--millisats-per-byte` for all occurrences.
> 
> The rest LGTM.

Done

File: cmd/tdex/config.go [hunk 1]
```diff
@@ -197,11 +197,11 @@ func configConnectAction(c *cli.Context) (err error) {
...
 		"proto":          proto,
-		"no_tls":         strconv.FormatBool(proto == "http"),
+		noTlsKey:         strconv.FormatBool(proto == "http"),
 		"rpcserver":      rpcServerAddr,
-		"no_macaroons":   noMacaroons,
-		"tls_cert_path":  tlsCertPath,
-		"macaroons_path": macaroonsPath,
+		noMacaroonsKey:   noMacaroons,
+		tlsCertPathKey:   tlsCertPath,
+		macaroonsPathKey: macaroonsPath,
```
Comment (altafan) [inline cmd/tdex/config.go]: revert these changes please, only the flags must change, not the content of the json config file

File: cmd/tdex/fee.go [hunk 2]
```diff
@@ -53,7 +47,7 @@ var (
 				Required: true,
 			},
 			&cli.Uint64Flag{
-				Name:  "millisatsperbyte",
+				Name:  "milli-sats-per-byte",
```
Comment (altafan) [inline cmd/tdex/fee.go]: `millisats-per-byte`

File: cmd/tdex/config.go (hunk 1 above)
Comment (altafan) [reply cmd/tdex/config.go]: also `config set` command must not be affected by changes

**Review by altafan — APPROVED** [review]
